ui/src/components/Canvas.tsx

```diff
@@ -531,18 +531,35 @@ const CodeNode = memo<Props>(function ({
     }
   }, [data.parent, setPodParent, id]);
 
+  const getPicture = async (pod) => {
+    const requesOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ code: pod.content }),
+      mode: "no-cors" as RequestMode,
+      accept: "image/png",
+    };
+    const res = await fetch("http://localhost:3099/api/cook", requesOptions);
+    const pic = await res.blob();
+    // navigator.clipboard.write([new ClipboardItem({ "image/png": pic })]);
```

Comment:
I think this `navigator.clipboard.write` works. I can paste the image to Google Slides:

<img width="900" alt="Screenshot 2022-12-26 at 6 19 08 PM" src="https://user-images.githubusercontent.com/4576201/209600796-8baaf334-ab77-4251-b08d-16e5b7eb3f6f.png">